### PR TITLE
Update Disable-PhysicalDiskIdentification Ex1 Cmds

### PR DESCRIPTION
**Pull request by katy-pan**
Update command as discovered by partner Bug 31760701. Change by ecosystem PM (Katy Pan) and approval by feature team software engineer (Taylor Hope)

### CODE REVIEW

File: docset/windows/storage/Disable-PhysicalDiskIdentification.md [hunk 1]
```diff
@@ -52,8 +52,7 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
 
 ### Example 1: Disable the identification LED on all physical disks in a pool
 ```
-PS C:\>$StPool = (Get-StoragePool -FriendlyName "SpacePool")
-PS C:\> Disable-PhysicalDiskIndication -StoragePool $StPool
+PGet-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
 ```
 
 The first command gets the storage poll named SpacePool by using the Get-StoragePool cmdlet.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/Disable-PhysicalDiskIdentification.md:58]:
```suggestion
```

Comment (JohanFreelancer9) [reply docset/windows/storage/Disable-PhysicalDiskIdentification.md:58]: @katy-pan - please confirm whether this line should be deleted, and if necessary, please commit the deletion.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/windows/storage/Disable-PhysicalDiskIdentification.md [hunk 2]
```diff
@@ -52,11 +52,9 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
 
 ### Example 1: Disable the identification LED on all physical disks in a pool
 ```
-PS C:\>$StPool = (Get-StoragePool -FriendlyName "SpacePool")
-PS C:\> Disable-PhysicalDiskIndication -StoragePool $StPool
+PGet-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
```
Comment (tseward) [inline docset/windows/storage/Disable-PhysicalDiskIdentification.md:55]: This should be in the format of:

```
PS C:\> Get-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
```

I'm assuming the "P" in front of "Get-" was a typo.

Where is $pool being initialized?

File: docset/windows/storage/Disable-PhysicalDiskIdentification.md [hunk 3]
```diff
@@ -52,11 +52,9 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
...
 ```
 
-The first command gets the storage poll named SpacePool by using the Get-StoragePool cmdlet.
 The command stores the pool in the $StPool variable.
```
Comment (tseward) [inline docset/windows/storage/Disable-PhysicalDiskIdentification.md:58]: Did you mean to modify this to indicate the variable "$pool"?

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by get-itips — COMMENTED** [review]
I am ok once @tseward suggestions are applied

File: docset/windows/storage/Disable-PhysicalDiskIdentification.md (hunk 3 above)
Comment (dariomws) [inline docset/windows/storage/Disable-PhysicalDiskIdentification.md:58]: This example disables the identification LED on all physical disks associated with the storage pool named "SpacePool".

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by e0i** [comment]
Hello, @katy-pan This PR is good to go once the formatting fixes have been applied. Thank you for this contribution.

File: docset/windows/storage/Disable-PhysicalDiskIdentification.md [hunk 4]
```diff
@@ -52,11 +52,9 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
 
 ### Example 1: Disable the identification LED on all physical disks in a pool
 ```
-PS C:\>$StPool = (Get-StoragePool -FriendlyName "SpacePool")
-PS C:\> Disable-PhysicalDiskIndication -StoragePool $StPool
+Get-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
```
Comment (tseward) [inline docset/windows/storage/Disable-PhysicalDiskIdentification.md:55]: This should be prefixed with `PS C:\>`.

**Review by tseward — CHANGES_REQUESTED** [review]

**Pull request merged**